src/cdtp/page.rs

```diff
@@ -13,14 +13,21 @@ pub struct Frame {
     pub unreachable_url: Option<String>,
 }
 
-/// The format a screenshot will be captured in
-#[derive(Debug, Clone, Serialize)]
+#[derive(Debug, Serialize)]
 #[serde(rename_all = "lowercase")]
-pub enum ScreenshotFormat {
+pub(crate) enum InternalScreenshotFormat {
```

Comment:
From my testing this doesn't seem to be necessary, I suppose because `cdtp` is already a public module in `lib.rs`?
```suggestion
pub enum InternalScreenshotFormat {
```